src/store/guestNameStore.js

```diff
@@ -20,86 +20,88 @@
  *
  */
 import Vue from 'vue'
+import { defineStore } from 'pinia'
+
+export const useGuestNameStore = defineStore('guestNameStore', {
+	state: () => ({
+		guestNames: {},
+	}),
 
-const state = {
-	guestNames: {
-	},
-}
 
-const getters = {
+	getters : {
 	/**
 	 * Gets the participants array
 	 *
 	 * @param {object} state the state object.
 	 * @return {Array} the participants array (if there are participants in the store)
 	 */
-	getGuestName: (state) => (token, actorId) => {
-		if (state.guestNames[token] && state.guestNames[token][actorId]) {
-			return state.guestNames[token][actorId]
+    getGuestName(state) {
+		return (token, actorId) => {
+		  if (state.guestNames[token] && state.guestNames[token][actorId]) {
+			return state.guestNames[token][actorId] 
+		  }
```

Comment:
```suggestion
    getGuestName(token, actorId) {
			return this.guestNames[token]?.[actorId]  ?? t('spreed', 'Guest')
	}
```

Let's simplify that

`state` and `getters` should be available with `this.`